### ISSUE TITLE
buildpack samples fail

### ISSUE THREAD

**Issue by trisberg** [knative/build-templates#51]
Trying the buildpack samples [1,2] and they both fail, I'm getting the following error in the logs:

```
2018/08/19 20:06:00 Error: failed to get droplet metadata: failed to failed to open ./result.json: open ./result.json: no such file or directory
```
[1] https://github.com/knative/docs/tree/master/serving/samples/buildpack-app-dotnet
[2] https://github.com/knative/docs/tree/master/serving/samples/buildpack-function-nodejs

**Comment by trisberg** [comment]
These are the logs from the `buildpck-app-dotnet` build:

```
$ kubectl logs --all-containers pod/buildpack-sample-app-00001-vtzt5
2018/08/20 13:23:51 Set StringFlag zap-logger-config to default: .
2018/08/20 13:23:51 Set StringFlag loglevel.creds-init to default: .
{"level":"error","ts":1534771431.3863323,"caller":"logging/config.go:42","msg":"Failed to parse the logging config. Falling back to default logger.","error":"empty logging configuration","build.knative.dev/jsonconfig":"","stacktrace":"github.com/knative/build/pkg/logging.NewLogger\n\t/go/src/github.com/knative/build/pkg/logging/config.go:42\ngithub.com/knative/build/pkg/logging.NewLoggerFromDefaultConfigMap\n\t/go/src/github.com/knative/build/pkg/logging/config.go:51\nmain.main\n\t/go/src/github.com/knative/build/cmd/creds-init/main.go:29\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:198"}
{"level":"info","ts":1534771431.3868523,"logger":"creds-init","caller":"creds-init/main.go:38","msg":"Credentials initialized."}
2018/08/20 13:23:52 Set StringFlag zap-logger-config to default: .
2018/08/20 13:23:52 Set StringFlag loglevel.git-init to default: .
{"level":"error","ts":1534771432.357731,"caller":"logging/config.go:42","msg":"Failed to parse the logging config. Falling back to default logger.","error":"empty logging configuration","build.knative.dev/jsonconfig":"","stacktrace":"github.com/knative/build/pkg/logging.NewLogger\n\t/go/src/github.com/knative/build/pkg/logging/config.go:42\ngithub.com/knative/build/pkg/logging.NewLoggerFromDefaultConfigMap\n\t/go/src/github.com/knative/build/pkg/logging/config.go:51\nmain.main\n\t/go/src/github.com/knative/build/cmd/git-init/main.go:56\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:198"}
{"level":"info","ts":1534771432.7607224,"logger":"git-init","caller":"git-init/main.go:74","msg":"Successfully cloned \"https://github.com/cloudfoundry-samples/dotnet-core-hello-world\" @ \"master\""}
-----> Dotnet-Core Buildpack version 2.1.2
-----> Supplying Dotnet Core
-----> Installing libunwind 1.2.1
       Download [https://buildpacks.cloudfoundry.org/dependencies/manual-binaries/dotnet/libunwind-1.2.1-linux-x64-80af276a.tgz]
       using the default SDK
-----> Installing dotnet 2.1.301
       Download [https://buildpacks.cloudfoundry.org/dependencies/dotnet/dotnet.2.1.301.linux-amd64-e4f7186f.tar.xz]
-----> Finalizing Dotnet Core
-----> Restore dotnet dependencies
         Restoring packages for /tmp/app/dotnet-core-hello-world.csproj...
         Installing Microsoft.Extensions.FileSystemGlobbing 2.0.1.
         Installing Microsoft.NETCore.Platforms 1.0.1.
         Installing System.Numerics.Vectors 4.4.0.
         Installing Microsoft.NETCore.DotNetAppHost 2.0.0.
         Installing System.Text.Encodings.Web 4.4.0.
         Installing Microsoft.AspNetCore.Http.Features 2.0.3.
         Installing System.Runtime.CompilerServices.Unsafe 4.4.0.
         Installing Microsoft.Extensions.FileProviders.Physical 2.0.1.
         Installing Microsoft.Extensions.Logging 2.0.2.
         Installing System.Diagnostics.DiagnosticSource 4.4.1.
         Installing Microsoft.Extensions.DependencyInjection 2.0.0.
         Installing System.Reflection.Metadata 1.5.0.
         Installing Microsoft.Extensions.Configuration.FileExtensions 2.0.2.
         Installing Microsoft.Extensions.Configuration.EnvironmentVariables 2.0.2.
         Installing Libuv 1.10.0.
         Installing Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions 2.0.4.
         Installing Microsoft.Extensions.Configuration.CommandLine 2.0.2.
         Installing Microsoft.AspNetCore.Server.IISIntegration 2.0.3.
         Installing Microsoft.NETCore.App 2.0.0.
         Installing Microsoft.AspNetCore.Server.Kestrel 2.0.4.
         Installing Microsoft.Extensions.Configuration 2.0.2.
         Installing Microsoft.Extensions.Logging.Abstractions 2.0.2.
         Installing Microsoft.AspNetCore.HttpOverrides 2.0.3.
         Installing System.Security.Principal.Windows 4.4.0.
         Installing Microsoft.Extensions.Options 2.0.2.
         Installing Microsoft.AspNetCore.Hosting.Abstractions 2.0.3.
         Installing Microsoft.AspNetCore.Http 2.0.3.
         Installing Microsoft.AspNetCore.Http.Extensions 2.0.3.
         Installing Microsoft.AspNetCore.Authentication.Core 2.0.3.
         Installing NETStandard.Library 2.0.0.
         Installing Microsoft.NETCore.DotNetHostPolicy 2.0.0.
         Installing Microsoft.AspNetCore.Server.Kestrel.Core 2.0.4.
         Installing Microsoft.NETCore.Platforms 2.0.0.
         Installing Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv 2.0.4.
         Installing Microsoft.Extensions.Configuration.Abstractions 2.0.2.
         Installing System.Threading.Tasks.Extensions 4.4.0.
         Installing Microsoft.AspNetCore.Hosting 2.0.3.
         Installing Microsoft.Extensions.DependencyInjection.Abstractions 2.0.0.
         Installing Microsoft.Extensions.Hosting.Abstractions 2.0.3.
         Installing Microsoft.Extensions.FileProviders.Abstractions 2.0.1.
         Installing Microsoft.Extensions.Primitives 2.0.0.
         Installing Microsoft.AspNetCore.Hosting.Server.Abstractions 2.0.3.
         Installing Microsoft.AspNetCore.Http.Abstractions 2.0.3.
         Installing Microsoft.Extensions.ObjectPool 2.0.0.
         Installing Microsoft.AspNetCore.WebUtilities 2.0.3.
         Installing Microsoft.Net.Http.Headers 2.0.3.
         Installing System.Buffers 4.4.0.
         Installing Microsoft.AspNetCore.Authentication.Abstractions 2.0.3.
         Installing Microsoft.NETCore.DotNetHostResolver 2.0.0.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.props.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.targets.
         Restore completed in 2.03 sec for /tmp/app/dotnet-core-hello-world.csproj.
       Required dotnetframework versions: [2.0.0]
-----> Installing dotnet-framework 2.0.0
       Download [https://buildpacks.cloudfoundry.org/dependencies/dotnet-framework/dotnet-framework.2.0.0.linux-amd64-13cb2a76.tar.xz]
       **WARNING** A newer version of dotnet-framework is available in this buildpack. Please adjust your app to use version 2.0.7 instead of version 2.0.0 as soon as possible. Old versions of dotnet-framework are only provided to assist in migrating to newer versions.
-----> Publish dotnet
       Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
       Copyright (C) Microsoft Corporation. All rights reserved.
       
         Restoring packages for /tmp/app/dotnet-core-hello-world.csproj...
         Installing Microsoft.NETCore.DotNetAppHost 2.0.7.
         Installing Microsoft.NETCore.DotNetHostResolver 2.0.7.
         Installing Microsoft.NETCore.DotNetHostPolicy 2.0.7.
         Installing NETStandard.Library 2.0.1.
         Installing Microsoft.NETCore.Platforms 2.0.2.
         Installing Microsoft.NETCore.App 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetAppHost 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.App 2.0.7.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.props.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.targets.
         Restore completed in 3.44 sec for /tmp/app/dotnet-core-hello-world.csproj.
         dotnet-core-hello-world -> /tmp/app/bin/Debug/netcoreapp2.0/ubuntu.14.04-x64/dotnet-core-hello-world.dll
         dotnet-core-hello-world -> /tmp/contents307356597/deps/0/dotnet_publish/
-----> Cleaning staging area
       Removing .nuget
       Removing .local
       Removing dotnet
2018/08/20 13:24:39 No matching credentials found for index.docker.io, falling back on anonymous
2018/08/20 13:24:40 Error: failed to get droplet metadata: failed to failed to open ./result.json: open ./result.json: no such file or directory
Error from server (BadRequest): container "nop" in pod "buildpack-sample-app-00001-vtzt5" is waiting to start: PodInitializing
```

**Comment by sclevine** [comment]
The `packs/cf` images haven't changed. Just looking at the output, it appears that the `droplet` volume mount is not being propagated between the steps. I'll try to debug this further.

**Comment by trisberg** [comment]
I tested with the release files for Knative 0.1.1 and the build works fine then with the following log output:

```
$ kubectl logs --all-containers pod/buildpack-sample-app-00001-hp2s4
-----> Dotnet-Core Buildpack version 2.1.2
-----> Supplying Dotnet Core
-----> Installing libunwind 1.2.1
       Download [https://buildpacks.cloudfoundry.org/dependencies/manual-binaries/dotnet/libunwind-1.2.1-linux-x64-80af276a.tgz]
       using the default SDK
-----> Installing dotnet 2.1.301
       Download [https://buildpacks.cloudfoundry.org/dependencies/dotnet/dotnet.2.1.301.linux-amd64-e4f7186f.tar.xz]
-----> Finalizing Dotnet Core
-----> Restore dotnet dependencies
         Restoring packages for /tmp/app/dotnet-core-hello-world.csproj...
         Installing Microsoft.NETCore.DotNetAppHost 2.0.0.
         Installing Microsoft.Extensions.FileSystemGlobbing 2.0.1.
         Installing Microsoft.NETCore.Platforms 1.0.1.
         Installing System.Numerics.Vectors 4.4.0.
         Installing System.Text.Encodings.Web 4.4.0.
         Installing Microsoft.AspNetCore.Http.Features 2.0.3.
         Installing System.Runtime.CompilerServices.Unsafe 4.4.0.
         Installing Microsoft.NETCore.DotNetHostResolver 2.0.0.
         Installing Microsoft.Extensions.FileProviders.Physical 2.0.1.
         Installing System.Reflection.Metadata 1.5.0.
         Installing System.Diagnostics.DiagnosticSource 4.4.1.
         Installing Microsoft.Extensions.Configuration.FileExtensions 2.0.2.
         Installing Microsoft.Extensions.Configuration.EnvironmentVariables 2.0.2.
         Installing Microsoft.Extensions.Logging 2.0.2.
         Installing Microsoft.Extensions.DependencyInjection 2.0.0.
         Installing Libuv 1.10.0.
         Installing Microsoft.Extensions.Configuration.CommandLine 2.0.2.
         Installing Microsoft.AspNetCore.Server.IISIntegration 2.0.3.
         Installing Microsoft.NETCore.App 2.0.0.
         Installing Microsoft.AspNetCore.Server.Kestrel 2.0.4.
         Installing Microsoft.Extensions.Configuration 2.0.2.
         Installing Microsoft.Extensions.Logging.Abstractions 2.0.2.
         Installing Microsoft.AspNetCore.HttpOverrides 2.0.3.
         Installing System.Security.Principal.Windows 4.4.0.
         Installing Microsoft.Extensions.Options 2.0.2.
         Installing Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions 2.0.4.
         Installing Microsoft.AspNetCore.Hosting.Abstractions 2.0.3.
         Installing System.Threading.Tasks.Extensions 4.4.0.
         Installing Microsoft.AspNetCore.Http.Extensions 2.0.3.
         Installing Microsoft.AspNetCore.Authentication.Abstractions 2.0.3.
         Installing Microsoft.AspNetCore.Http 2.0.3.
         Installing Microsoft.AspNetCore.WebUtilities 2.0.3.
         Installing Microsoft.AspNetCore.Authentication.Core 2.0.3.
         Installing Microsoft.Extensions.ObjectPool 2.0.0.
         Installing Microsoft.AspNetCore.Server.Kestrel.Core 2.0.4.
         Installing Microsoft.Net.Http.Headers 2.0.3.
         Installing System.Buffers 4.4.0.
         Installing Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv 2.0.4.
         Installing Microsoft.AspNetCore.Hosting 2.0.3.
         Installing Microsoft.AspNetCore.Http.Abstractions 2.0.3.
         Installing Microsoft.NETCore.Platforms 2.0.0.
         Installing Microsoft.AspNetCore.Hosting.Server.Abstractions 2.0.3.
         Installing NETStandard.Library 2.0.0.
         Installing Microsoft.Extensions.FileProviders.Abstractions 2.0.1.
         Installing Microsoft.Extensions.Hosting.Abstractions 2.0.3.
         Installing Microsoft.Extensions.Primitives 2.0.0.
         Installing Microsoft.Extensions.DependencyInjection.Abstractions 2.0.0.
         Installing Microsoft.Extensions.Configuration.Abstractions 2.0.2.
         Installing Microsoft.NETCore.DotNetHostPolicy 2.0.0.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.props.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.targets.
         Restore completed in 1.91 sec for /tmp/app/dotnet-core-hello-world.csproj.
       Required dotnetframework versions: [2.0.0]
-----> Installing dotnet-framework 2.0.0
       Download [https://buildpacks.cloudfoundry.org/dependencies/dotnet-framework/dotnet-framework.2.0.0.linux-amd64-13cb2a76.tar.xz]
       **WARNING** A newer version of dotnet-framework is available in this buildpack. Please adjust your app to use version 2.0.7 instead of version 2.0.0 as soon as possible. Old versions of dotnet-framework are only provided to assist in migrating to newer versions.
-----> Publish dotnet
       Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
       Copyright (C) Microsoft Corporation. All rights reserved.
       
         Restoring packages for /tmp/app/dotnet-core-hello-world.csproj...
         Installing Microsoft.NETCore.DotNetAppHost 2.0.7.
         Installing Microsoft.NETCore.DotNetHostResolver 2.0.7.
         Installing Microsoft.NETCore.DotNetHostPolicy 2.0.7.
         Installing NETStandard.Library 2.0.1.
         Installing Microsoft.NETCore.Platforms 2.0.2.
         Installing Microsoft.NETCore.App 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetAppHost 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.App 2.0.7.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.props.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.targets.
         Restore completed in 2.74 sec for /tmp/app/dotnet-core-hello-world.csproj.
         dotnet-core-hello-world -> /tmp/app/bin/Debug/netcoreapp2.0/ubuntu.14.04-x64/dotnet-core-hello-world.dll
         dotnet-core-hello-world -> /tmp/contents439581403/deps/0/dotnet_publish/
-----> Cleaning staging area
       Removing .nuget
       Removing .local
       Removing dotnet
2018/08/20 14:13:33 No matching credentials found for index.docker.io, falling back on anonymous
2018/08/20 14:13:43 pushed blob sha256:21324a9f04e76c93078f3a782e3198d2dded46e4ec77958ddd64f701aecb69c0
2018/08/20 14:13:43 pushed blob sha256:d2411d24e5bd840e72708575c3a84ac0cbc01ebdc5447afb1fb7a0f24f63e195
2018/08/20 14:13:44 pushed blob sha256:1124eb40dd68654b8ca8f5d9ec7e439988a4be752a58c8f4e06d60ab1589abdb
2018/08/20 14:13:44 pushed blob sha256:a5733e6358eec8957e81b1eb93d48ef94d649d65c69a6b1ac49f616a34a74ac1
2018/08/20 14:13:44 pushed blob sha256:beaf4fac18c58420c4d6d69e14a611120bb42d6cce80a5084bb9c223d74fa8bf
2018/08/20 14:13:54 pushed blob sha256:6be38da025345ffb57d1ddfcdc5a2bc052be5b9491825f648b49913d51e41acb
2018/08/20 14:13:54 gcr.io/cf-spring-funkytown/buildpack-sample-app:latest: digest: sha256:2c2367106e5fdddb8334c8ed3bdcf8f6249123b6febb2d9cd3336ca7c0249c4d size: 1083
Nothing to push
Aruba:java-function-invoker trisberg$ kubectl logs --all-containers pod/buildpack-sample-app-00001-hp2s4
-----> Dotnet-Core Buildpack version 2.1.2
-----> Supplying Dotnet Core
-----> Installing libunwind 1.2.1
       Download [https://buildpacks.cloudfoundry.org/dependencies/manual-binaries/dotnet/libunwind-1.2.1-linux-x64-80af276a.tgz]
       using the default SDK
-----> Installing dotnet 2.1.301
       Download [https://buildpacks.cloudfoundry.org/dependencies/dotnet/dotnet.2.1.301.linux-amd64-e4f7186f.tar.xz]
-----> Finalizing Dotnet Core
-----> Restore dotnet dependencies
         Restoring packages for /tmp/app/dotnet-core-hello-world.csproj...
         Installing Microsoft.NETCore.DotNetAppHost 2.0.0.
         Installing Microsoft.Extensions.FileSystemGlobbing 2.0.1.
         Installing Microsoft.NETCore.Platforms 1.0.1.
         Installing System.Numerics.Vectors 4.4.0.
         Installing System.Text.Encodings.Web 4.4.0.
         Installing Microsoft.AspNetCore.Http.Features 2.0.3.
         Installing System.Runtime.CompilerServices.Unsafe 4.4.0.
         Installing Microsoft.NETCore.DotNetHostResolver 2.0.0.
         Installing Microsoft.Extensions.FileProviders.Physical 2.0.1.
         Installing System.Reflection.Metadata 1.5.0.
         Installing System.Diagnostics.DiagnosticSource 4.4.1.
         Installing Microsoft.Extensions.Configuration.FileExtensions 2.0.2.
         Installing Microsoft.Extensions.Configuration.EnvironmentVariables 2.0.2.
         Installing Microsoft.Extensions.Logging 2.0.2.
         Installing Microsoft.Extensions.DependencyInjection 2.0.0.
         Installing Libuv 1.10.0.
         Installing Microsoft.Extensions.Configuration.CommandLine 2.0.2.
         Installing Microsoft.AspNetCore.Server.IISIntegration 2.0.3.
         Installing Microsoft.NETCore.App 2.0.0.
         Installing Microsoft.AspNetCore.Server.Kestrel 2.0.4.
         Installing Microsoft.Extensions.Configuration 2.0.2.
         Installing Microsoft.Extensions.Logging.Abstractions 2.0.2.
         Installing Microsoft.AspNetCore.HttpOverrides 2.0.3.
         Installing System.Security.Principal.Windows 4.4.0.
         Installing Microsoft.Extensions.Options 2.0.2.
         Installing Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions 2.0.4.
         Installing Microsoft.AspNetCore.Hosting.Abstractions 2.0.3.
         Installing System.Threading.Tasks.Extensions 4.4.0.
         Installing Microsoft.AspNetCore.Http.Extensions 2.0.3.
         Installing Microsoft.AspNetCore.Authentication.Abstractions 2.0.3.
         Installing Microsoft.AspNetCore.Http 2.0.3.
         Installing Microsoft.AspNetCore.WebUtilities 2.0.3.
         Installing Microsoft.AspNetCore.Authentication.Core 2.0.3.
         Installing Microsoft.Extensions.ObjectPool 2.0.0.
         Installing Microsoft.AspNetCore.Server.Kestrel.Core 2.0.4.
         Installing Microsoft.Net.Http.Headers 2.0.3.
         Installing System.Buffers 4.4.0.
         Installing Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv 2.0.4.
         Installing Microsoft.AspNetCore.Hosting 2.0.3.
         Installing Microsoft.AspNetCore.Http.Abstractions 2.0.3.
         Installing Microsoft.NETCore.Platforms 2.0.0.
         Installing Microsoft.AspNetCore.Hosting.Server.Abstractions 2.0.3.
         Installing NETStandard.Library 2.0.0.
         Installing Microsoft.Extensions.FileProviders.Abstractions 2.0.1.
         Installing Microsoft.Extensions.Hosting.Abstractions 2.0.3.
         Installing Microsoft.Extensions.Primitives 2.0.0.
         Installing Microsoft.Extensions.DependencyInjection.Abstractions 2.0.0.
         Installing Microsoft.Extensions.Configuration.Abstractions 2.0.2.
         Installing Microsoft.NETCore.DotNetHostPolicy 2.0.0.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.props.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.targets.
         Restore completed in 1.91 sec for /tmp/app/dotnet-core-hello-world.csproj.
       Required dotnetframework versions: [2.0.0]
-----> Installing dotnet-framework 2.0.0
       Download [https://buildpacks.cloudfoundry.org/dependencies/dotnet-framework/dotnet-framework.2.0.0.linux-amd64-13cb2a76.tar.xz]
       **WARNING** A newer version of dotnet-framework is available in this buildpack. Please adjust your app to use version 2.0.7 instead of version 2.0.0 as soon as possible. Old versions of dotnet-framework are only provided to assist in migrating to newer versions.
-----> Publish dotnet
       Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
       Copyright (C) Microsoft Corporation. All rights reserved.
       
         Restoring packages for /tmp/app/dotnet-core-hello-world.csproj...
         Installing Microsoft.NETCore.DotNetAppHost 2.0.7.
         Installing Microsoft.NETCore.DotNetHostResolver 2.0.7.
         Installing Microsoft.NETCore.DotNetHostPolicy 2.0.7.
         Installing NETStandard.Library 2.0.1.
         Installing Microsoft.NETCore.Platforms 2.0.2.
         Installing Microsoft.NETCore.App 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetAppHost 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy 2.0.7.
         Installing runtime.linux-x64.Microsoft.NETCore.App 2.0.7.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.props.
         Generating MSBuild file /tmp/app/obj/dotnet-core-hello-world.csproj.nuget.g.targets.
         Restore completed in 2.74 sec for /tmp/app/dotnet-core-hello-world.csproj.
         dotnet-core-hello-world -> /tmp/app/bin/Debug/netcoreapp2.0/ubuntu.14.04-x64/dotnet-core-hello-world.dll
         dotnet-core-hello-world -> /tmp/contents439581403/deps/0/dotnet_publish/
-----> Cleaning staging area
       Removing .nuget
       Removing .local
       Removing dotnet
2018/08/20 14:13:33 No matching credentials found for index.docker.io, falling back on anonymous
2018/08/20 14:13:43 pushed blob sha256:21324a9f04e76c93078f3a782e3198d2dded46e4ec77958ddd64f701aecb69c0
2018/08/20 14:13:43 pushed blob sha256:d2411d24e5bd840e72708575c3a84ac0cbc01ebdc5447afb1fb7a0f24f63e195
2018/08/20 14:13:44 pushed blob sha256:1124eb40dd68654b8ca8f5d9ec7e439988a4be752a58c8f4e06d60ab1589abdb
2018/08/20 14:13:44 pushed blob sha256:a5733e6358eec8957e81b1eb93d48ef94d649d65c69a6b1ac49f616a34a74ac1
2018/08/20 14:13:44 pushed blob sha256:beaf4fac18c58420c4d6d69e14a611120bb42d6cce80a5084bb9c223d74fa8bf
2018/08/20 14:13:54 pushed blob sha256:6be38da025345ffb57d1ddfcdc5a2bc052be5b9491825f648b49913d51e41acb
2018/08/20 14:13:54 gcr.io/cf-spring-funkytown/buildpack-sample-app:latest: digest: sha256:2c2367106e5fdddb8334c8ed3bdcf8f6249123b6febb2d9cd3336ca7c0249c4d size: 1083
Nothing to push
```

**Comment by dprotaso** [comment]
Try again as this has landed https://github.com/knative/build-templates/pull/53

**Comment by trisberg** [comment]
Fixed by #53

/close
